.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Personally, I'd rather not check this in right now. Thoughts, @edwjchen ?

Reply:
I don't think this should be checked in, we can add it to the .gitignore for now?